src/renderer/cursor_renderer/mod.rs

```diff
@@ -86,6 +90,11 @@ impl Corner {
             self.t = 0.0;
             self.start_position = self.current_position;
             self.previous_destination = destination;
+            self.length_multiplier = if settings.distance_length_adjust {
+                (destination - self.current_position).length().log10()
```

Comment:
Interesting. Can you explain the log term?

Reply:
That's solely so that the multiplier doesn't get too large. If I had just let it scale linearly for instance with my setting for the animation time(0.035) it would have taken about 0.07 for just regular typing but 6.3s(!) to go from top to bottom. log seemed like a reasonable choice, though other functions might work as well(roots etc.).

Reply:
Makes good sense to me. I will try this out on my local machine and get it merged tonight. Great idea!